src/apps/project-home/AssignmentsView/Wizard/Progress/ProgressUpdating.tsx

```diff
@@ -104,7 +104,7 @@ export const ProgressUpdating = (props: ProgressUpdatingProps) => {
         // - check for added read only layers on new documents
         for (let i = 0; i < documentChanges.added.length; i++) {
           const doc = documentChanges.added[i];
-          const prevDoc = previous.documents.find((d) => (d.id = doc.id));
+          const prevDoc = previous.documents.find((d) => d.id === doc.id);
```

Comment:
Such a classic :-D (Having fond memories of doing the same thing at various times ;-)